packages/contentful/src/utils.ts

```diff
@@ -120,3 +121,20 @@ export const updateEntryFields = (
   });
   return updatedEntry;
 };
+
+export const patchAndPublish = async (
+  entry: Entry,
+  fields: Record<string, unknown>,
+): Promise<void> => {
+  const patch: OpPatch[] = Object.entries(fields).map(([key, value]) => ({
```

Comment:
you can avoid adding the json-patch library to bundle by doing 
```suggestion
  const patch: Parameters<Entry["patch"]>[0] = Object.entries(fields).map(([key, value]) => ({
```